sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -104,13 +104,13 @@ interface FileManager {
    * Deletes an item
    * @param path item path to delete
    */
-  deleteItem(path: string): Promise<boolean>;
+  delete(path: string): Promise<boolean>;
 
   /*
    * Copies an item from source to destination. Item name collisions at the
    * destination are handled by backend.
    * @param itemPath path to item to copy
```

Comment:
change docname (itemPath) to match function (path)
As with save, should this return Promise<boolean>?